eng/pipelines/corefx-base.yml

```diff
@@ -78,11 +78,14 @@ jobs:
 
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
+          - _archiveTestsParameter: /p:ArchiveTests=Tests
```

Comment:
Did you do this change in case we want to override this parameter for perf tests and integration tests later on? Adding an extra filter?

Reply:
Discussed offline, I guess 😁